core/dbt/context/providers.py

```diff
@@ -710,6 +710,10 @@ def _get_namespace_builder(self):
             self.model,
         )
 
+    @contextproperty
+    def invocation_args_to_dict(self):
```

Comment:
Nit: I slightly prefer
```suggestion
    def invocation_args_dict(self):
```